Schlange.py

```diff
@@ -39,7 +39,7 @@ class KeywordTranslator:
         "selbst" : "self",
         "drucke" : "print",
         "mathe" : "math",
-        "Wurzel" : "sqrt"
+        "wurzel" : "sqrt"

```

Comment:
Noticed this was also changed in @AntonPieper's commit: [a89c7fd](https://github.com/actopozipc/German-Python-Interpreter/pull/1/commits/a89c7fdc404d88427862bcbc42e6ee57e660714b#diff-d9e78e6195c8416fa498aef198b60384be105893b476190278cf92a596efed14R42)